packages/language-server/src/plugins/typescript/TypeScriptPlugin.ts

```diff
@@ -329,6 +330,31 @@ export class TypeScriptPlugin
         const { lang, tsDoc } = await this.getLSAndTSDoc(document);
         const mainFragment = tsDoc.getFragment();
 
+        const cssClassHelper = new CSSClassDefinitionLocator(tsDoc, position, document);
```

Comment:
The idea of the plugin folders is that they are mostly independent of each other, so it would be better to put this logic into the CSSPlugin or move the class into the TypeScript plugin folder - but I feel like the best place would be `SveltePlugin.ts`. This brings us to a question to which I don't have a good answer yet: What is the best place for code like this? It's cross-cutting so it's not obvious to me in which plugin this does belong. Maybe this hints at that the plugin system needs a redesign / needs to be replaced with something new, but that's for another day.

Reply:
Feel free to relocate wherever you feel is best